plugin/PDFView.qml

```diff
@@ -132,6 +136,35 @@ SilicaFlickable {
         linkColor: Theme.highlightColor;
         pagePlaceholderColor: Theme.highlightColor;
 
+        onPageLayoutChanged: if (rememberPositionConfig.value) restorePageSettings()
```

Comment:
For the pageLayoutChanged signal, the issue is that, on documentLoad signal, the page geometry is not yet computed. Indeed, the signal for the page geometry is pageLayout. But as you mentioned, this is emitted on each resizing (read here, portrait / landscape rotation). So we need a signal that is firstPageLayout. Since, I found emitting such a signal quite ugly because too much oriented for the page-remember feature, I decide to plug the pageRestore function on every pageLayoutChanged *and* in the function do the job only once, thanks to the "if (_settings !== undefined)" statement.